src/flows/shared/steps/OrganizeCollection/Editor/StagingArea.tsx

```diff
@@ -48,6 +39,22 @@ const IMAGE_SIZES: Record<number, number> = {
   6: 78,
 };
 
+function _getDndWidth(numImagesInRow: number) {
+  return (
+    IMAGE_SIZES[numImagesInRow] * numImagesInRow + PADDING_BETWEEN_STAGED_IMAGES_PX * numImagesInRow
+  );
+}
+
+// Width of DND area for each Column # setting
+const DND_WIDTHS: Record<number, number> = {
+  1: _getDndWidth(1),
+  2: _getDndWidth(2),
+  3: _getDndWidth(3),
+  4: _getDndWidth(4),
+  5: _getDndWidth(5),
+  6: _getDndWidth(6),
+};
```

Comment:
now we're using exact measurements; the editor should always be aligned / perfectly centered